tests/resources/existing-resources/main.tf

```diff
@@ -0,0 +1,29 @@
+module "resource_group" {
+  source                       = "terraform-ibm-modules/resource-group/ibm"
+  version                      = "1.1.6"
+  resource_group_name          = var.resource_group == null ? "${var.prefix}-resource-group" : null
+  existing_resource_group_name = var.resource_group
+}
+
+module "cos" {
+  source                 = "terraform-ibm-modules/cos/ibm"
+  version                = "8.9.1"
+  cos_instance_name      = "${var.prefix}-cos"
+  kms_encryption_enabled = false
+  retention_enabled      = false
+  resource_group_id      = module.resource_group.resource_group_id
+  bucket_name            = "${var.prefix}-cb"
+}
+
+module "create_scc_instance" {
+  source                            = "terraform-ibm-modules/scc/ibm"
+  version                           = "1.7.2"
```

Comment:
SCC module is local to this repo, so you should use a local source. Please be aware that when you do that, you may need to update the test itself, same way we did in https://github.com/terraform-ibm-modules/terraform-ibm-secrets-manager/pull/117

Reply:
addressed